### PR TITLE
Powershell NuGet fix + version updates

### PR DESCRIPTION
**Pull request by codingupastorm**

### CODE REVIEW

**Comment by bokobza** [comment]
what about Stratis.Bitcoin.Features.SmartContracts? Does it have a version?

File: src/Stratis.Bitcoin.Features.SmartContracts/Stratis.Bitcoin.Features.SmartContracts.csproj [hunk 1]
```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <TargetFramework>netcoreapp2.1</TargetFramework>
 	<CodeAnalysisRuleSet>..\None.ruleset</CodeAnalysisRuleSet>
+	<Version>0.0.2-beta</Version>
```
Comment (codingupastorm) [inline src/Stratis.Bitcoin.Features.SmartContracts/Stratis.Bitcoin.Features.SmartContracts.csproj]: @bokobza this?

Comment (bokobza) [reply src/Stratis.Bitcoin.Features.SmartContracts/Stratis.Bitcoin.Features.SmartContracts.csproj]: lol sorry mate

**Comment by bokobza** [comment]
We need the RuntimeObserver too apparently.

**Pull request merged**